spring-cloud-netflix-eureka-client/src/test/java/org/springframework/cloud/netflix/eureka/EurekaClientAutoConfigurationTests.java

```diff
@@ -425,16 +466,6 @@ public void eurekaClientClosed() {
 		}
 	}
 
-	@Test
-	public void eurekaRegistrationClosed() throws IOException {
```

Comment:
This test can't be removed as it is a regression

Reply:
test is being moved to ```shouldCloseDiscoveryClient``` since `EurekaRegistration` is not closing DiscoveryClient anymore